doc/sections/commands.md

```diff
@@ -57,13 +57,13 @@ modifier or an exposition marker. In the following example, `const` is a
 modifier and `private` is an exposition marker.
 
 ```
-symbol const Nat : TYPE
-symbol const zero : Nat
-symbol const succ (x:Nat) : Nat
+const symbol Nat : TYPE
+const symbol zero : Nat
+const symbol succ (x:Nat) : Nat
 symbol add : Nat ⇒ Nat ⇒ Nat
-symbol const list : Nat ⇒ TYPE
-symbol const nil : List zero
-symbol const cons : Nat ⇒ ∀n, List n ⇒ List(succ n)
+const symbol list : Nat ⇒ TYPE
+const symbol nil : List zero
+const symbol cons : Nat ⇒ ∀n, List n ⇒ List(succ n)
 private symbol aux : ∀n, List n ⇒ Nat
```

Comment:
`constant`, not `const`.

Reply:
Fixed in 28a70b39044793af24c859077a1aac32331171ee.